packages/flutter_tools/lib/src/isolated/sdk_web_configuration.dart

```diff
@@ -16,7 +16,7 @@ class SdkWebConfigurationProvider extends SdkConfigurationProvider {
   SdkWebConfigurationProvider(this._artifacts);
 
   final Artifacts _artifacts;
-  late SdkConfiguration _configuration;
+  SdkConfiguration _configuration;
```

Comment:
Should this `SdkConfiguration?` ?

Reply:
I'm going to be rebasing this change on Chris's change when it lands, so I won't be making any changes to this file